src/components/charts/BorderShutdown.js

```diff
@@ -0,0 +1,208 @@
+import React from "react"
+import Grid from "@material-ui/core/Grid"
+import Paper from "@material-ui/core/Paper"
+import Typography from "@material-ui/core/Typography"
+import outboundIcon from "@components/icons/outbound.png"
+import inboundIcon from "@components/icons/inbound.png"
+import { getCountryFromISO } from "@/utils/mapBaiduCountry"
+import { withStyles } from "@material-ui/core/styles"
+import Tooltip from "@material-ui/core/Tooltip"
+import styled from "styled-components"
+import { useTranslation } from "react-i18next"
+
+const StyledTooltip = withStyles({
+  tooltip: {
+    backgroundColor: "white",
+    color: "rgba(0, 0, 0, 0.87)",
+    fontSize: 16,
+    borderRadius: 14,
+    padding: 10,
+  },
+})(Tooltip)
+
+const Chip = styled(Paper)`
+  color: white;
+  width: auto;
+  height: auto;
+  border-radius: 17pt;
+  cursor: pointer;
+  box-shadow: none;
+  background-color: rgba(255, 255, 255, 0.12);
+  padding: 5px 10px;
+  &:hover {
+    background-color: rgba(255, 255, 255, 0.9);
+    color: black;
+  }
+`
```

Comment:
It's better to use [MaterialUi/Chip](https://material-ui.com/zh/components/chips/) or styled('div') if need to override these much style at Paper.